### ISSUE TITLE
simpleJson endpoint is not presented by API (or at least is difficult to find!)

### ISSUE THREAD

**Issue by ben-lavelle** [axa-group/Parsr#497]
### Summary 

The simpleJson output described in the [docs](https://github.com/axa-group/Parsr/blob/master/docs/simple-json-output.md) looks to be a useful output in simple use-cases. However, it is not implemented in the python client, and doesn't appear to be implemented in the API (or is at least it beats me how to find it!).

I can't see any other issues or PRs that reference simpleJson; apologies if this is a duplicate (or straight-up wrong!)

### Steps To Reproduce

This seems to me to be regardless of what config file and input file is used, so the Jupyter notebook example with `defaultConfig.json` is sufficient to demonstrate.

Steps to reproduce the behaviour:

1. Run the Jupyter notebook example as [here](https://github.com/axa-group/Parsr/blob/master/demo/parsr-jupyter-demo/notebook.ipynb).
2. Send a document, e.g. the sample document provided, with `defaultConfig.json` or any config with simpleJson output requested and check other outputs are as expected.
3. Call e.g. `parsr.get_status()` and note that even if requested, the endpoint for `simpleJson` output is not listed in the response (example request hash):

```txt
{'request_id': '01cecaa5bd8624aa9d8a1d899dde73',
 'server_response': {'id': '01cecaa5bd8624aa9d8a1d899dde73',
  'json': '/api/v1/json/01cecaa5bd8624aa9d8a1d899dde73',
  'csv': '/api/v1/csv/01cecaa5bd8624aa9d8a1d899dde73',
  'text': '/api/v1/text/01cecaa5bd8624aa9d8a1d899dde73',
  'markdown': '/api/v1/markdown/01cecaa5bd8624aa9d8a1d899dde73'}}
```

4. Visit other endpoints with e.g. (substituting request hash)

```txt
http://localhost:3001/api/v1/json/01cecaa5bd8624aa9d8a1d899dde73
```

5. Find no such endpoint for `/api/v1/simplejson` or `/api/v1/simple-json`, `api/v1/simple/json` etc.

6. See in the API logs that the simpleJson output **_was_** in fact produced, e.g.

```txt
[2020-10-08T15:43:10] INFO  (parsr-api/7 on 9da727d13778): Exporting markdown...
[2020-10-08T15:43:10] INFO  (parsr-api/7 on 9da727d13778): Writing file: /opt/app-root/src/api/server/dist/output/sampleFile-01cecaa5bd8624aa9d8a1d899dde73/sampleFile.simple.json
[2020-10-08T15:43:11] INFO  (parsr-api/7 on 9da727d13778): Writing file: /opt/app-root/src/api/server/dist/output/sampleFile-01cecaa5bd8624aa9d8a1d899dde73/sampleFile.txt
[2020-10-08T15:43:11] INFO  (parsr-api/7 on 9da727d13778): Exporting markdown...
[2020-10-08T15:43:11] INFO  (parsr-api/7 on 9da727d13778): Writing file: /opt/app-root/src/api/server/dist/output/sampleFile-01cecaa5bd8624aa9d8a1d899dde73/sampleFile.md
```

but fails to be presented as a discoverable endpoint.

### Expected behavior
An endpoint should be available for the useful simpleJson output. It'd also be nice to add a `get_simpleJson()` method to ParsrClient or add `parsr.get_json([simple: bool=False])` option to the existing full JSON method.

### Actual behavior
No endpoint found at some sensible addresses; no way that was apparent to me of accessing the simpleJSON output. Correspondingly no way to access simpleJSON from the py client.

### Environment

- Reference commit/version: `Parsr==v1.2.0, parsr-client==3.0.0`
- Other platform details: `Docker Engine - Community v19.03.5`
- Hardware: Mac air 2015
- OS: MacOS High Sierra

### Additional context
I would be happy to put together the py client implementation as a PR if you think it is a useful feature, but my knowledge of the API in js is absolutely minimal so I'm not sure I can fix this part. For the same reason sorry if this is a daft issue and I'm missing something obvious!

**Comment by agarwal-nitesh** [comment]
@ben-lavelle Did you end up extracting the file from the location (container/host) or did you find a way to hook an endpoint that serves the simple JSON?

**Comment by agarwal-nitesh** [comment]
Hi @kleag. I built using the docker-compose-build file and the API worked but some staged changes were not part of the above commit and are required. Here they are https://github.com/axa-group/Parsr/pull/593. My apologies.

**Comment by kleag** [comment]
Don't apologize @agarwal-nitesh . In the contrary, thanks a lot for your contribution and help! With your clone, I was able to build and use the docker composed images. 

I had to copy `./docker/parsr-base/policy.xml` to the root of the project and to create a user with UID 1001 on my system (my user is 1000) to make it work.

I also had to add `"simpleJson":true` in the `config.json` file. It would be more coherent if it would be `simple-json` as in the endpoint. 
The status message should also contain an entry for the `simple-json` result. 

Maybe the two above points should be new issues in fact.
